Ginger/GingerCoreNET/Run/RunsetExecutor.cs

```diff
@@ -281,6 +281,11 @@ private void CopyCustomizedVariableConfigurations(VariableBase customizedVar, Va
             RepositoryItemBase.ObjectsDeepCopy(customizedVar, originalVar);//need to replace 'ObjectsDeepCopy' with AutoMapper and to map on it which values should be overiden
             originalVar.DiffrentFromOrigin = customizedVar.DiffrentFromOrigin;
             originalVar.MappedOutputVariable = customizedVar.MappedOutputVariable;
+            //Fix for Bug 9310 - Beeline - Empty variable are not being saved in Run Configuration
```

Comment:
we don't add ticket numbers or bugs in the comment. 
You can explain what the particular piece of code is doing if it is needed but not the tickets